synapse/config/emailconfig.py

```diff
@@ -509,6 +474,62 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
             % DEFAULT_SUBJECTS
         )
 
+    def read_templates(
+        self,
+        filenames: List[str],
+        custom_template_directory: Optional[str] = None,
+        autoescape: bool = True,
+        filters: Dict[str, Callable] = {},
+    ) -> List[jinja2.Template]:
+        """Load a list of template files from disk using the given variables and filters.
+
+        This function will attempt to load the given templates from the default Synapse
+        template directory. If `custom_template_directory` is supplied, that directory
+        is tried first.
+
+        Files read are treated as Jinja templates. These templates are not rendered yet.
+
+        Args:
+            filenames: A list of template filenames to read.
+
+            custom_template_directory: A directory to try to look for the templates
+                before using the default Synapse template directory instead.
+
+            autoescape: Whether to automatically escape template variables when the templates
+                are rendered.
+
+            filters: Custom functions which can modify the given template variables upon
+                render, if the template file specifies them.
```

Comment:
I referenced https://jinja.palletsprojects.com/en/master/api/#writing-filters to ensure this was all sane. 👍